tests/system/providers/databricks/example_databricks_sensors.py

```diff
@@ -0,0 +1,111 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This is an example DAG which uses the DatabricksSqlSensor,
+DatabricksPartitionSensor and DatabricksTableChangesSensor.
+The first task checks for a generic SQL statement against a Delta table,
+and if a result is returned, the task succeeds, else it times out.
+The second task checks for the specified partitions' presence in a
+Delta table. If it exists, the task succeeds, else it times out.
+The third task checks for data related changes in a Delta
+table via versions. If the version retrieved from hive metastore
+is different from the version stored in Airflow metadata, the DAG
+succeeds, else it times out.
+"""
+from __future__ import annotations
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.providers.databricks.sensors.partition import DatabricksPartitionSensor
+from airflow.providers.databricks.sensors.sql import DatabricksSqlSensor
+from airflow.providers.databricks.sensors.table_changes import DatabricksTableChangesSensor
+
+# [docs]
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+# [docs]
+DAG_ID = "example_databricks_sensor"
+
+with DAG(
+    dag_id=DAG_ID,
+    schedule="@daily",
+    start_date=datetime(2021, 1, 1),
+    tags=["example"],
+    catchup=False,
+) as dag:
+    # [docs]
+    connection_id = "databricks_default"
+    sql_endpoint_name = "Starter Warehouse"
+
+    # [START howto_sensor_databricks_sql]
```

Comment:
These `START/END` markers are used to include code snippets in guides (generally). It would be great if there was accompanying documentation for these new sensors and ones that take advantage of the snippets being outlined in this DAG. There are a lot of examples in operator guides on how this is done throughout the providers.